api/user/CourseProgress.ts

```diff
@@ -0,0 +1,40 @@
+export const fetchCoursesProgress = async (
+  id: string,
+  courseId: string,
+  topicId: string,
+  sequentialTopic:string,
+  lessonId: string,
+  videoId: string,
+  sequentialLesson:string
+) => {
+    try {
+    const response = await fetch(
+      `${process.env.NEXT_PUBLIC_BACKEND_URL}/course/courseProgress
+            `,
+      {
+        method: "PUT",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          id: id,
+          courseId: courseId,
+          topicId: topicId,
+          sequentialTopic:sequentialTopic,
+          lessonId: lessonId,
+          videoId: videoId,
+          sequentialLesson:sequentialLesson,
+          viewVideo: true,
+        }),
+      }
+    );
+
+    if (response.ok) {
+      console.log("Video information sent successfully");
```

Comment:
Do you think it's necessary to let in the console a log when the request is successful?